### PR TITLE
[SEDONA-122] add ST_GeomFromWKB overload for BYTES column containing (e)wkb byte array.

### PR DESCRIPTION
**Pull request by Elephantusparvus**
## Did you read the Contributor Guide?

- Yes, I have read [Contributor Rules](https://sedona.apache.org/community/rule/) and [Contributor Development Guide](https://sedona.apache.org/community/develop/)

## Is this PR related to a JIRA ticket?

- Yes, the URL of the assoicated JIRA ticket is https://issues.apache.org/jira/browse/SEDONA-122. The PR name follows the format `[SEDONA-122] my subject`.

## What changes were proposed in this PR?
Reading geometry object from BYTES column containing (e)wkb data and added a simple unit test. Maybe some parts (table creation) of the test should be refactored to the TestBase, might be useful later.

## How was this patch tested?
I added a very simple unit test with one Linestring to the ConstructorTests. The wkb case in general currently has no other tests.

## Did this PR include necessary documentation updates?
Not sure on this one.

### CODE REVIEW

File: flink/src/test/java/org/apache/sedona/flink/ConstructorTest.java [hunk 1]
```diff
@@ -30,6 +30,17 @@
 import static org.apache.flink.table.api.Expressions.call;
 import static org.junit.Assert.assertEquals;
 
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.common.typeinfo.BasicTypeInfo;
+import org.apache.flink.api.common.typeinfo.PrimitiveArrayTypeInfo;
+import org.apache.flink.api.java.typeutils.RowTypeInfo;
+
+import org.apache.flink.streaming.api.datastream.DataStream;
+
```
Comment (netanel246) [inline flink/src/test/java/org/apache/sedona/flink/ConstructorTest.java:39]: Please remove unused spaces.

**Review by netanel246 — CHANGES_REQUESTED** [review]
This looks Great!
Please add the changes also to the Spark SQL module (with docs and test):
https://github.com/apache/incubator-sedona/blob/21cfa42e34e769e2dcc326d98bbcfd964f540d74/sql/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/Constructors.scala#L229

**Comment by Elephantusparvus** [comment]
Sorry, for not replying. Just wanted to let you know that I will do the needed additions/fixes, but I am currently having a tight schedule and it might take a few days, until I find time again.

File: flink/src/test/java/org/apache/sedona/flink/ConstructorTest.java (hunk 1 above)
Comment (Elephantusparvus) [reply flink/src/test/java/org/apache/sedona/flink/ConstructorTest.java:38]: Should be fixed with [3def393](https://github.com/apache/incubator-sedona/pull/628/commits/3def393bbcef98ba19651814a60a362e797a4b36).

**Comment by jiayuasu** [comment]
@Elephantusparvus Thank you. Looking forward to it!

**Review by jiayuasu — APPROVED** [review]

**Pull request merged**